lms/urls.py

```diff
@@ -145,7 +145,7 @@
     url(r'^api/experiments/', include('experiments.urls', namespace='api_experiments')),
 
     # [COLARAZ_CUSTOM] colaraz urls
-    url(r'^api/colaraz/', include('openedx.features.colaraz_features.urls')),
+    url(r'^api/colaraz/', include('openedx.features.colaraz_features.urls', namespace='colaraz_features')),
```

Comment:
Can we change '^api/colaraz/' to '^colaraz/' as its pointing to whole app, not just the apis